src/views/datatable110.blade.php

```diff
@@ -4,12 +4,12 @@
         oTable = jQuery('#{{ $id }}').DataTable({
             "processing": true,
             "serverSide": true,
-            "ajax": "/",
+            "ajax": "{{ isset($options['route']) ? $options['route'] : '/' }}",
```

Comment:
We should add a separate option for this instead of putting this into the options.
The options are normally used for something else that is not implemented yet :/